### PR TITLE
[stable/sonatype-nexus] Add resource requests and limits support for nexus

### PR DESCRIPTION
**Pull request by Miouge1**
**What this PR does / why we need it**:
Nexus proxy sidecar container has no configurable resources
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/sonatype-nexus/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.4.4
+version: 1.4.5
```
Comment (rjkernick) [inline stable/sonatype-nexus/Chart.yaml]: Please bump the minor version instead as this adds a new feature

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/sonatype-nexus/Chart.yaml (hunk 1 above)
Comment (Miouge1) [reply stable/sonatype-nexus/Chart.yaml]: OK, here you go `1.5.0` in latest commit.

**Review by rjkernick — APPROVED** [review]

**Pull request merged**